libseccomp/src/syscall.rs

```diff
@@ -158,16 +159,7 @@ impl ScmpSyscall {
     /// # Ok::<(), Box<dyn std::error::Error>>(())
     /// ```
     pub fn from_name_by_arch(name: &str, arch: ScmpArch) -> Result<Self> {
-        let name_c = CString::new(name)?;
-        let nr = unsafe { seccomp_syscall_resolve_name_arch(arch.to_sys(), name_c.as_ptr()) };
-        if nr == __NR_SCMP_ERROR {
-            return Err(SeccompError::with_msg(format!(
-                "Could not resolve syscall name {}",
-                name
-            )));
-        }
-
-        Ok(Self { nr })
+        Self::from_name_by_impl(seccomp_syscall_resolve_name_arch as _, name, arch)
```

Comment:
I think `as _` is not required anymore.

Reply:
Returned unresolved state:
@rusty-snake  Did you update something?

Reply:
Turns out you have to `git add` your changes. :smile: